### PR TITLE
Feature/get cars

### PR DESCRIPTION
**Pull request by zxander-D**
added get all cars test EMPTY

change build gradle to generate rest docs

added index.adoc for rest documentation

### CODE REVIEW

File: src/test/java/com/galvanize/alpha/speedwaytrials/racecars/RaceCarIT.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+@Transactional
+@AutoConfigureMockMvc
+@AutoConfigureRestDocs
+public class RaceCarIT {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Test
+    public void getAllCars_EmptyList() throws Exception {
```
Comment (davidgrieser) [inline src/test/java/com/galvanize/alpha/speedwaytrials/racecars/RaceCarIT.java:27]: Nice baseline test

File: src/main/java/com/galvanize/alpha/speedwaytrials/racecar/RaceCarController.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.util.List;
+
+@RestController
+@RequestMapping("cars")
+public class RaceCarController {
+
+    @GetMapping
+    public String getAllCars(){
+        return "[]";
```
Comment (davidgrieser) [inline src/main/java/com/galvanize/alpha/speedwaytrials/racecar/RaceCarController.java:15]: simple direct implementation.

File: build.gradle [hunk 3]
```diff
@@ -1,45 +1,53 @@
...
+    inputs.dir snippetsDir
+    attributes 'snippets': snippetsDir
+    dependsOn test
+}
+
+bootJar {
+    dependsOn asciidoctor
+    from("${asciidoctor.outputDir}/html5") {
+        into 'static/docs'
+    }
```
Comment (davidgrieser) [inline build.gradle:52]: This is the one thing I wish start.spring.io would do by default.

**Review by davidgrieser — APPROVED** [review]
Not sure if this resolves a particular issue or not. If so you need to mention it in the Description or as a comment using the hash `#` symbol followed by a number.

Some formatting changes in `build.gradle`, but otherwise the change to add REST support is needed. Might be out of scope for this PR, but y'all can decide on that.

**Review by j-tanz — APPROVED** [review]
LGTM

**Pull request merged**